app/src/main/java/dev/arkbuilders/arkmemo/graphics/SVG.kt

```diff
@@ -0,0 +1,215 @@
+package dev.arkbuilders.arkmemo.graphics
+
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Path as AndroidDrawPath
+import android.util.Xml
+import dev.arkbuilders.arkmemo.ui.viewmodels.DrawPath
+import org.xmlpull.v1.XmlPullParser
+import java.nio.file.Path
+import java.util.Stack
+import kotlin.io.path.reader
+import kotlin.io.path.writer
+
+class SVG {
+    private var strokeColor = "black"
+    private var fill = "none"
+    private var viewBox = "0 0 100 100"
```

Comment:
Ideally, we should have dedicated integer variables for each of 4 (x,y,w,h) but probably not worth right now.

@tuancoltech do you think that's fine for now?

Reply:
@kirillt @ShubertMunthali I agree with Kirillt.

I think having `x, y, w, h` defined as parameters not only improve readability of this code, but also enhancing extendability, imagining the case that later on we want to pass in `x, y` into the `SVG` object as well.
Right now I don't understand what the first two params mean, honestly.

We should probably also have an API for building the `viewBox` string from the 4 defined parameters if we decide to go with this approach. I believe that helps with

Reply:
The parameters are ```x, y, width, height``` as treated by svg standard for a viewbox.
I agree with you guys about creating separate variables for these params. The same should go with stroke and fill.